src/targets/gpu/CMakeLists.txt

```diff
@@ -112,6 +116,10 @@ if(NOT MIGRAPHX_USE_COMPOSABLEKERNEL)
             ${CMAKE_CURRENT_SOURCE_DIR}/jit/ck_gemm_softmax_gemm.cpp)
 endif()
 
+if (MIGRAPHX_USE_ROCBLAS)
+    set(ROCBLAS_SRCS rocblas.cpp gemm_impl.cpp)
```

Comment:
You already ifdef these sources, so we dont need to remove them from the list in cmake.